sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_options.hpp

```diff
@@ -1330,15 +1330,4 @@ namespace Azure { namespace Storage { namespace Blobs {
     BlobAccessConditions AccessConditions;
   };
 
-  /**
-   * @brief Optional parameters for BlobBatchClient::SubmitBatch.
-   */
-  struct SubmitBlobBatchOptions
```

Comment:
If this was public, it needs to include a changelog note about this breaking change. 
it´s a nit, I guess, since no one is using something useless, but I think the right thing would be to call it

Reply:
I added an item to changelog in PR #1309. I removed everything related to blob batch except this one in that PR.